source/AMDSMainCommon.h

```diff
@@ -14,10 +14,14 @@ namespace AMDSMain_Common {
 	/*
 	 * Print the usage of the application
 	 */
-	void printUsage(const QString &appName)
+	void printUsage(const QString &appName, const QString &errMsg="")
 	{
+		if (errMsg != "") {
+			qDebug() << "\nERROR: " << errMsg << "\n";
+		}
+
 		qDebug() << "\nUsages:\n"
-				 << QString("	%1 --intf=[interface name] [--port=[portnumber]]\n").arg(appName)
+				 << QString("	%1 --intf=[interface name] [--port=[portnumber]] [--cfg=[cfgFileName]]\n").arg(appName)
```

Comment:
The bracketing here seems to suggest that the --cfg is optional, but it isn't it doesn't seem to be in usage (line 39). Is this correct?